docs/analyze.rst

```diff
@@ -0,0 +1,383 @@
+Analysis of Scenario Data
+-------------------------
+Several categories of analysis focusing on:
+
+- transmission congestion/utilization
+- generation and generator capacity
+- emission (:math:`\rm{CO}_2`, :math:`\rm{NO}_x` and :math:`\rm{SO}_2`) from thermal
+  generators
+- curtailment of renewable generators
+
+can be conducted with the PostREISE package.
+
+You will find below some code snippets that will help you use these analysis tools
+for each category. Information on the input and returned parameter(s) of the analysis
+functions can be found on this website. Use the :ref:`modindex` to locate a module and
+access the documentation along with the source code of the objects therein.
+
+
+Transmission
+++++++++++++
+- calculate the hourly congestion surplus
+
+  .. code-block:: python
+
+      from powersimdata.scenario.scenario import Scenario
```

Comment:
Should we use `from powersimdata import Scenario` for the code snippets?

Reply:
Done